lib/src/models/mixins/inboxable.dart

```diff
@@ -51,4 +51,8 @@ abstract class InboxableMixin implements RedditBaseInitializedMixin {
   ///
   /// This method pertains only to objects which were retrieved via the inbox.
   Future<void> uncollapse() async => await reddit.inbox.uncollapse([this]);
+
+  /// Removes the message from the recipient's view of their inbox.
+  Future<void> remove() async =>
+      await reddit.post(apiPath['del_msg'], {'id': fullname});
```

Comment:
Typically it's a good idea to provide types for `Map` and `List` since they'll default to having `dynamic` as their type arguments when nothing else is provided. I've been bitten by issues related to this before, so it's better to be safe than sorry!

```suggestion
      await reddit.post(apiPath['del_msg'], <String, String>{'id': fullname});
```